src/test/java/HomeWork21.java

```diff
@@ -0,0 +1,33 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class HomeWork21 extends BaseTest {
+
+
+
+    @Test
+    public static void renamePlaylist() {
+        login("valentineguts@gmail.com", "te$t$tudent");
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)